.idea/nadlanks-bot.iml

```diff
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
```

Comment:
please delte .idea dictionary. find way how to cancel the auto addining of this dict